.github/workflows/unit-tests.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
   tests:
     strategy:
       matrix:
```

Comment:
Could it be simplified? it is a matrix with a single element

Reply:
Can we do this in a new PR ?